stdlib/public/SDK/Foundation/Foundation.swift

```diff
@@ -567,11 +567,12 @@ extension Array : _ObjectiveCBridgeable {
 
 extension NSDictionary : ExpressibleByDictionaryLiteral {
   public required convenience init(
-    dictionaryLiteral elements: (NSCopying, AnyObject)...
+    dictionaryLiteral elements: (_Hashable, Any)...
```

Comment:
This use of `_Hashable` can't be replaced with a generic init, right?  If so, we can't use `_Hashable` here, because it is just a temporary workaround that will go away (once we have generic subscripts), and then this API will be left without a type to use.